## Notable Projects Portfolio
<br> 
This portfolio showcases a selection of computational biology projects, including finalized and published works as well as ongoing research.
<br> 
<br> 
<div style="display: flex; align-items: center;">
    <a href="https://ch.linkedin.com/in/sean-o-toole-662a16176" style="display: flex; align-items: center;">
        <img src="https://github.com/sean-otoole/portfolio/blob/main/images/linkedIn_logo.png" width="25" height="25" style="margin-right: 8px;">
        In case you want to connect!
    </a>
</div>
<div style="display: flex; align-items: center;">
    <a href="https://scholar.google.com/citations?user=KXvfb1kAAAAJ&hl=en&oi=ao" style="display: flex; align-items: center;">
        <img src="https://github.com/sean-otoole/portfolio/blob/main/images/google_scholar_logo.png" width="25" height="25" style="margin-right: 8px;">
        Take a look at my published works!
    </a>
</div>

## Education
- <img align="left" width="25" height="25" src="https://github.com/sean-otoole/portfolio/blob/main/images/brandeis_seal.png">Ph.D. Brandeis University, Neuroscience
- <img align="left" width="25" height="25" src="https://github.com/sean-otoole/portfolio/blob/main/images/WPI_seal.png">B.S. W.P.I., Molecular Biology and Biotechnology
  

## Projects


<img align="left" width="250" height="150" src="https://github.com/sean-otoole/portfolio/blob/main/images/BRCA_2024.png"> **[Spatial transcriptomics/Breast cancer project, 2024:](https://github.com/sean-otoole/BRCA_2024)**
This ongoing project (i.e. not finalized) utilizes spatial transcriptomics data sets to examine the degree to which specific B-cell subtypes may contribute to the differences between pre- and post-menopausal women with breast cancer. I am examining how specific gene signatures and cell types vary between patient samples. I am experimenting with segregating tumors based on the degree of immune cell invasiveness and using image segmentation to restrict the analysis strictly to tumor tissue. Additionally, I am exploring gene signatures with an area under the curve (AUC) approach, in which the AUC scores are calculated for each gene signature of interest. Furthermore, cell type annotation for each spot is performed using a logistic regression model (CellTypist), and I verify the results of the cell type assignment by examining the cosine similarity scores of each location compared to a previously annotated dataset.
#

<img align="left" width="250" height="150" src="https://github.com/sean-otoole/scell_otoole_2023/blob/main/images/heatmaps_main.png"> **[Single-cell RNA-sequencing analysis for O'Toole, 2023:](https://github.com/sean-otoole/scell_otoole_2023)**
In this project, I analyzed single-cell RNA sequencing data I had collected to identify the transcriptional profiles of functional groups of neocortical neurons. I constructed a virtual reality setup in which mouse neuronal populations were labeled in vivo as being highly active or inactive during the presentation of specific stimuli. I then dissociated the photolabeled cells and FACS sorted them for single-cell RNA sequencing. The analysis pipeline I constructed relied on integrating approximately 60,000 neurons acquired during these experiments with a higher-read depth, publicly available, dataset to identify specific neuronal subtypes. After confidently assigning subtypes, I could use a bootstrap analysis to determine which populations were enriched as a function of specific behavioral stimuli. The work employed numerous single-cell RNA sequencing techniques such as dimensionality reduction, matrix factorization, clustering, and bootstrapping.
#

<img align="left" width="250" height="150" src="https://github.com/sean-otoole/2p_neural_activity_otoole_2023/blob/main/images/fig_1_clip.png"> **[Neural activity and behavioral analysis for O'Toole, 2023:](https://github.com/sean-otoole/2p_neural_activity_otoole_2023)**
This repository, developed in MATLAB, comprises two distinct sets of analyses, both aimed at investigating the neural activity patterns of specific cell types. The analyses primarily involve manipulating a metadata file from a two-photon imaging pipeline. The activity levels of different neuronal groups are compared during specific behavioral events. I employ a combination of hierarchical nested bootstrap and regression analysis to compare these populations.
<br>
#

<img align="left" width="250" height="150" src="https://github.com/sean-otoole/bulk_RNA_seq_otoole_2023/blob/main/images/fig_s5_clip.png"> **[Bulk RNA-sequencing analysis of AAVs for O'Toole, 2023:](https://github.com/sean-otoole/bulk_RNA_seq_otoole_2023)**
In this analysis, utilizing a combination of R and Python, I compare the RNA expression patterns of various neuronal populations infected with adeno-associated viruses (AAVs). I map RNA-sequencing reads to a customized version of the mouse genome, which I have modified to account for AAV expression, and analyze transcriptional markers to assess the specificity of populations labeled with different viruses.
<br>

#

<img align="left" width="250" height="150" src="https://github.com/sean-otoole/aav_histology_otoole_2023/blob/main/images/figure_5.png"> **[Comparing the spatial patterns of AAVs for O'Toole, 2023:](https://github.com/sean-otoole/aav_histology_otoole_2023)**
In this project, I analyze the spatial expression of adeno-associated viruses (AAVs) in neocortical tissue slices. The images are processed using an ImageJ macro, and the final comparisons and computations are conducted with Python.
<br>
<br>
<br>

#

<img align="left" width="250" height="150" src="https://github.com/sean-otoole/single_nuclei_seq_widmer_2022/blob/main/otoole_widmer_clip.png"> **[single-nuclei RNA-sequencing analysis for Widmer, 2022:](https://github.com/sean-otoole/single_nuclei_seq_widmer_2022)** 
This code was written to ask the degree to which specific gene is knocked down across various cell types, after AAV infection.
<br>
<br>
<br>
<br>


#



<br />

<!--

-->

